### ISSUE TITLE
Roadmap for compatibility with other Corona apps?

### ISSUE THREAD

**Issue by bauxi** [austrianredcross/meta#10]
Since there are many Corona apps in many countries currently being developed and pretty much all rely on the Google/Apple contact tracing framework, what is the roadmap for collaboration with these other countries? Is it planned that these different apps are compatible?
I mean due to border-less Europe this would make pretty much sense.

**Comment by bastianh** [comment]
The apps are already compatible in a way that you can have multiple installed and you can send your contacts to different countries, but each country has different ways to check if an infection had happend. ( for example you might need to scan a qrcode from a test result )

While all those apps use the same frameworks from Apple/Google they are still quite different. For example the austrian app is (as far as I know) the only app that has the „yellow“ warning type for being probably sick. There are also other differences, so different countries do different calculations to decide how risky a contact with a infected person was depending on duration of the contact and the signal strength that was measured. 

There are also rules from apple/google about what you are allowed to do with the data and they are quite strictly enforced. I guess there are also different laws in different countries so I don‘t think this will happen.

From my point of view it would be much easier if everybody would just install the same app ,)

**Comment by bauxi** [comment]
I agree, from my point of view it would have been much easier if there was a European app or something like that.

Just to add to this discussion: This [article](https://www.derstandard.at/story/2000120043544/was-wurde-aus-der-stopp-corona-app) says that the German app is now able to communicate with 5 different countries - so it could be possible. And it actually could be quite helpful during the cold season.
